### PR TITLE
Update atom types of Pharmagkb pathway dataset

### PR DESCRIPTION
**Pull request by tanksha**
Updates atom type for
- Pharmagkb pathway
- Molecules such as Chebi, Uniprot and Pubchem
- Go terms

### CODE REVIEW

File: pharmagkb.py [hunk 1]
```diff
@@ -151,7 +161,8 @@ def generate_locations(elem, ns, chemical_nodes, pathway_id):
                match = go_location_re.match(location) 
                if match:
                    go_location = match.group(1)
-                   location_node = CConceptNode(go_location)
+                   go_type = find_gons.find_go_type(go_location)
```
Comment (noskill) [inline pharmagkb.py:164]: find_gons doesn't have find_go_type function

Comment (tanksha) [reply pharmagkb.py:164]: sorry, I forgot to commit. I just did.

File: find_gons.py [hunk 2]
```diff
@@ -31,4 +24,19 @@ def request_api(go_term):
     if result.ok:
```
Comment (noskill) [inline find_gons.py:24]: It would be useful to add a timeout to request and check the status to distinguish a network failure from 'not found' answer.

Comment (tanksha) [reply find_gons.py:24]: I have updated it, Please check it out. Thanks

**Comment by noskill** [comment]
Please merge. The code looks good to me.

**Pull request merged**